apps/gallery/js/info.js

```diff
@@ -44,18 +52,90 @@ function showFileInformation(fileinfo) {
           args: args
         },
         'info-type': {raw: fileinfo.type},
-        'info-date': {raw: MediaUtils.formatDate(fileinfo.date)},
+        'info-date': {raw: formattedDate},
         'info-resolution': {
           raw: fileinfo.metadata.width + 'x' + fileinfo.metadata.height
         }
       };
+      // placeholders for Exif. Also hide the exif fields.
+      exifFields.forEach((elem) => {
+        data[elem] = {raw: ''};
+        setFieldVisibility(elem, false);
+      });
 
       // Populate info overlay view
       MediaUtils.populateMediaInfo(data);
+
+      LazyLoader.load(['shared/js/media/jpeg-exif.js'], () => {
```

Comment:
We need to parse exif only for images, may be we should put this block of code inside a check if not fileinfo.metadata.video